test/material_snackbar_test.dart

```diff
@@ -0,0 +1,121 @@
+@Tags(const ['aot'])
+@TestOn('browser')
+import 'dart:async';
+import 'package:angular2/angular2.dart';
+import 'package:angular_test/angular_test.dart';
+import 'package:skawa_components/src/components/material_snackbar/material_snackbar.dart';
+import 'package:test/test.dart';
+import 'package:pageloader/html.dart';
+import 'package:pageloader/src/annotations.dart';
+
+@AngularEntrypoint()
+main() {
+  tearDown(disposeAnyRunningTest);
+  group('MaterialSnackbar |', () {
+    test('initializing', () async {
+      final bed = new NgTestBed<MaterialSnackbarTestComponent>();
+      final fixture = await bed.create();
+      final pageObject = await fixture.resolvePageObject(TestPO);
+      expect(pageObject.materialSnackbar, isNotNull);
+    });
+
+    test('displays a message (default duration: 3 sec)', () async {
+      final bed = new NgTestBed<MaterialSnackbarTestComponent>();
+      final fixture = await bed.create();
+      final pageObject = await fixture.resolvePageObject(TestPO);
+      await fixture.update((testElement) {
+        testElement.showMessage('hello');
+      });
+      await new Future.delayed(new Duration(milliseconds: 1000), () async {
+        await fixture.update();
+        await fixture.query<MaterialSnackbarComponent>((element) {
+          return element.componentInstance is MaterialSnackbarComponent;
+        }, (MaterialSnackbarComponent snackbar) {
+          expect(snackbar.message.text, 'hello');
+          expect(snackbar.show, true);
+        });
+        expect(pageObject.materialSnackbar.messageContainer.classes, mayEmit('show'));
+        print('snackbar shows up');
```

Comment:
instead of printing, should create another test case, we should never make any logic step after an `expect`

Reply:
done